chia/_tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -468,21 +470,20 @@ async def test_get_farmed_amount(wallet_environments: WalletTestFramework) -> No
     env = wallet_environments.environments[0]
     wallet_rpc_client = env.rpc_client
 
-    get_farmed_amount_result = await wallet_rpc_client.get_farmed_amount()
+    get_farmed_amount_result = await wallet_rpc_client.get_farmed_amount(GetFarmedAmount())
     get_timestamp_for_height_result = await wallet_rpc_client.get_timestamp_for_height(
         GetTimestampForHeight(uint32(3))
     )  # genesis + 2
 
-    expected_result = {
-        "blocks_won": 2,
-        "farmed_amount": 4_000_000_000_000,
-        "farmer_reward_amount": 500_000_000_000,
-        "fee_amount": 0,
-        "last_height_farmed": 3,
-        "last_time_farmed": get_timestamp_for_height_result.timestamp,
-        "pool_reward_amount": 3_500_000_000_000,
-        "success": True,
```

Comment:
Wonder what happened to this one? (I suspect you've added `| None` to a signature)